README.rst

```diff
@@ -42,6 +42,9 @@ Configuration
 ``config_sub_paths`` (default is ``[]``) specifies sub paths under which the mypy configuration file may be found.
     For each directory searched for the mypy config file, this also searches the sub paths specified here
 
+``report_progress`` (default is ``False``) report basic progress to the LSP client.
+    With this option, pylsp-mypy will report when mypy is running, given you editor supports LSP progress reporting. For small files this might produce annoying flashing in your editor, espacially in with ``live_mode``. For large projects, enabling this can be helpfull to assure yourself whether mypy is still running.
```

Comment:
```suggestion
    With this option, pylsp-mypy will report when mypy is running, given your editor supports LSP progress reporting. For small files this might produce annoying flashing in your editor, especially in with ``live_mode``. For large projects, enabling this can be helpful to assure yourself whether mypy is still running.
```